ci/pipelines/apps-test-ecr.yml

```diff
@@ -117,6 +125,7 @@ resources:
     icon: docker
     source:
       repository: govukpay/toolbox
+      variant: release
```

Comment:
This is matching on the suffix, correct? `123-release` and so on?

Reply:
It took me a minute to suss this out as well. Why couldn't they have just called this particular key "tag_suffix" or something?

Reply:
yep - see https://github.com/concourse/registry-image-resource#source-configuration